capellambse/svg/generate.py

```diff
@@ -65,8 +65,9 @@ def __init__(
         self,
         metadata: DiagramMetadata,
         objects: cabc.Sequence[ContentsDict],
+        **kwargs: dict[str, t.Any],
     ) -> None:
-        self.drawing = Drawing(metadata)
+        self.drawing = Drawing(metadata, **kwargs)
```

Comment:
The constant dict-packing and -unpacking doesn't make sense, and somewhat breaks with how the same API works over in the `diagram.Diagram`. IMO we should either just pass it around as a single dict, or unpack it into regular keyword arguments.